client/pages/Login/Invitation/Invitation.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useEffect } from 'react'
+import { useHistory } from 'react-router-dom'
+
+import { useAppDispatch, useAppSelector } from '@client/store'
+import { LoginActions } from '@client/store/login'
+import { Urls } from '@client/utils'
+import { useUser } from '@client/store/user'
+import { useTranslation } from 'react-i18next'
+
+import { Objects } from '@core/utils'
+import { BasePaths } from '@client/basePaths'
+
+const Invitation: React.FC = () => {
+  const dispatch = useAppDispatch()
+  const history = useHistory()
+  const invitedUser = useAppSelector((state) => state.login.invitedUser)
```

Comment:
create hook in `store/login/hooks` and export it in `store/login/index`
`store/login/hooks/index.ts` : `export const useInvitedUser = () => useAppSelector((state) => state.login.invitedUser)`